src/test/java/org/jabref/logic/formatter/bibtexfields/CleanupUrlFormatterTest.java

```diff
@@ -29,6 +29,12 @@ void extractURLFormLink() {
                 formatter.format("away.php?to=http%3A%2F%2Fwikipedia.org&a=snippet"));
     }
 
+    @Test
+    void validURLUnmodified() {
+        // the caller has to pay attention that this does not happen
```

Comment:
What should not happen? Maybe just remove the comment.

The call seems IMHO legal.

Reply:
That's on me, I copy-pasted and forgot to remove it.